allennlp/commands/elmo.py

```diff
@@ -0,0 +1,203 @@
+"""
+The ``elmo`` subcommand allows you to make bulk ELMo predictions.
```

Comment:
Can you add:

1) A TODO to add the link to the paper.
2) The output of `python -m allennlp.run elmo --help`, as with the docstrings of the other commands.

Reply:
I struggled with adding the output as a docstring, but I couldn't get it to work.  In fact, I couldn't even copy the code block from the predict docstring--make docs kept failing.

```
Traceback (most recent call last):
  File "/Users/michaels/miniconda3/envs/allennlp/lib/python3.6/site-packages/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/Users/michaels/miniconda3/envs/allennlp/lib/python3.6/site-packages/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/Users/michaels/miniconda3/envs/allennlp/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/Users/michaels/miniconda3/envs/allennlp/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 268, in build
    self.warn(*warning, **kwargs)
  File "/Users/michaels/miniconda3/envs/allennlp/lib/python3.6/site-packages/sphinx/application.py", line 399, in warn
    raise SphinxWarning(warntext)
sphinx.errors.SphinxWarning: /Users/michaels/hack/allenai/allennlp/allennlp/commands/elmo.py:docstring of allennlp.commands.elmo:1: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 27 supplied.

.. code-block:: bash
   $ python -m allennlp.run elmo --help
   usage: python -m allennlp.run [command] elmo [-h] [--vocab-path VOCAB_PATH]
                                                [--options-file OPTIONS_FILE]
                                                [--weight-file WEIGHT_FILE]
                                                [--batch-size BATCH_SIZE]
                                                [--cuda-device CUDA_DEVICE]
                                                [--use-sentence-key]
                                                input_file output_file

   Create word vectors using ELMo.

   positional arguments:
     input_file            path to input file
     output_file           path to output file

   optional arguments:
     -h, --help            show this help message and exit
     --vocab-path VOCAB_PATH
                           A path to a vocabulary file to generate
     --options-file OPTIONS_FILE
                           The path to the ELMo options file.
     --weight-file WEIGHT_FILE
                           The path to the ELMo weight file.
     --batch-size BATCH_SIZE
                           The batch size to use.
     --cuda-device CUDA_DEVICE
                           The cuda_device to run on.
     --use-sentence-key
```

Reply:
Pretty sure you need a blank line after the `.. code-block:: bash` line.  Sphinx is really finicky...

Reply:
That did it--although I swear I tried that previously and it didn't work.

Reply:
Done.